### PR TITLE
Update installation.md (make the steps more clear)

### PR DESCRIPTION
**Pull request by filiphosko**
I tried to make it more clear that there are two popular ways of starting up with React, my solution is not sophisticated by any means, just tried to propose a simple (temporary) solution that could do the trick. My PR is a reply to this issue https://github.com/facebook/react/issues/9275 .

### CODE REVIEW

File: docs/docs/installation.md [hunk 1]
```diff
@@ -20,7 +20,9 @@ If you're just interested in playing around with React, you can use CodePen. Try
 
 If you prefer to use your own text editor, you can also <a href="/react/downloads/single-file-example.html" download="hello.html">download this HTML file</a>, edit it, and open it from the local filesystem in your browser. It does a slow runtime code transformation, so don't use it in production.
 
-## Creating a Single Page Application
+Currently there are two popular ways to get started with React:
+
+## 1. Using Create React App
```
Comment (aweary) [inline docs/docs/installation.md]: I feel like numbering it makes it seem like these are two steps to `Creating a Single Page Application`, which makes it less clear what these instructions are saying 🤔

**Comment by gaearon** [comment]
Does anyone want to do the tab suggestion in the issue? I think it would be amazing.

File: docs/docs/installation.md (hunk 1 above)
Comment (filiphosko) [inline docs/docs/installation.md]: You are right, that's why I removed the line that says `Creating a Single Page Application` and instead added the line `Currently there are two popular ways to get started with React:`. Of course numbering isn't always the best solution, it was just my suggestion for a quick 'fix' ;)

**Comment by filiphosko** [comment]
@gaearon I can do the tabs version. How do you imagine the 'captions' for the tabs? React Native uses the tabs for the platforms, so they are short, but I don't know if we could avoid having large tabs (with captions 'Add React to an Existing Application', 'Use Create React App'). Or maybe it doesn't matter and I'm just too picky :)

**Comment by gaearon** [comment]
We went with tabs instead (https://github.com/facebook/react/pull/9401).
Sorry I didn't reply to your earlier comment!

I hope it's better now.

**Pull request closed without merging**